internal/cluster/validator.go

```diff
@@ -533,6 +535,11 @@ func (v *clusterValidator) networkPrefixValid(c *clusterPreprocessContext) (Vali
 	if !validationStatusToBool(clusterCidrDefined) {
 		return ValidationPending, "The Cluster Network CIDR is undefined."
 	}
+
+	if v.skipNetworkHostPrefixCheck(c) {
+		return ValidationSuccess, "The Cluster Network prefix is valid."
+	}
+
 	validClusterNetworks := funk.Filter(c.cluster.ClusterNetworks, func(clusterNetwork *models.ClusterNetwork) bool {
 		return clusterNetwork != nil &&
 			network.VerifyNetworkHostPrefix(clusterNetwork.HostPrefix) == nil &&
```

Comment:
`VerifyNetworkHostPrefix` verifies that the requested number of hosts matches the network parameters.  Even if we assume that each host has 1 address, can we verify that the cluster network is enough for the requested number of hosts?

Reply:
Do you mean VerifyClusterCidrSize() where that verification is done?  
I could move skipNetworkHostPrefixCheck() back into the loop here https://github.com/openshift/assisted-service/blob/master/internal/cluster/validator.go#L538 and check clusterNetworkPrefix for each cluster against the number of hosts, returning false to not skip, to let VerifyClusterCidrSize() return the error. Is that OK?

Reply:
@ori-amizur - I added back in check to ensure that cidr validation is done, let me know if that looks reasonable. Thanks.